static/src/javascripts/es6/projects/common/modules/crosswords/main.js

```diff
@@ -342,13 +342,22 @@ class Crossword extends React.Component {
         }
     }
 
+    clueIsInFocusGroup (clue) {
+        if (this.state.cellInFocus) {
+            const cluesForCell = this.cluesFor(this.state.cellInFocus.x, this.state.cellInFocus.y);
+            return _.contains(cluesForCell[this.state.directionOfEntry].group, clue.id);
+        } else {
+            return null;
+        }
+    }
+
     cluesData () {
         return _.map(this.props.data.entries, (entry) => ({
             entry: entry,
             hasAnswered: _.every(helpers.cellsForEntry(entry), (position) => {
                 return /^[A-Z]$/.test(this.state.grid[position.x][position.y].value);
             }),
-            isSelected: this.clueInFocus() === entry
+            isSelected: this.clueIsInFocusGroup(entry)
```

Comment:
You might be able to re-use `isHighlighted` (in this file), or re-factor it to share code with this. It is responsible for highlighting cells on the grid that are in the current group.